.circleci/export-e2e-test-user-vars.sh

```diff
@@ -7,27 +7,27 @@ for arg in "$@"
 do
   case "$arg" in
     "perf")
-       echo "export PERF_READER=$PUPPETEER_READER_PERF" >> $BASH_ENV
-       echo "export PERF_WRITER=$PUPPETEER_WRITER_PERF" >> $BASH_ENV
-       echo "export PERF_COLLABORATOR=$PUPPETEER_COLLABORATOR_PERF" >> $BASH_ENV
+       echo "export READER_USER=$PUPPETEER_READER_PERF@perf.fake-research-aou.org" >> $BASH_ENV
```

Comment:
same user variable names for all env (perf, staging, test, local) are going to break tests in other env.

Reply:
Are the base usernames different across environments - is that the issue?

I would just hardcode them here if you need to - there's no reason to be using Circle env vars anymore for this. Or probably better, if this could live in the puppeteer config code (that may be a larger refactoring).

Reply:
Noope, I was wrong. Changes are good.

Reply:
I'd like them to be hard coded here or puppeteer config. But I do not have permission to view circle configs. Probably need someone help to pull configs from circle.

Reply:
Can not view the full name, but just from the suffix, I think base usernames are different(
![Screen Shot 2021-06-25 at 11 07 11 AM](https://user-images.githubusercontent.com/6351731/123445028-80134a00-d5a5-11eb-82cc-921ea63dc778.png)
)